src/identifiers/logic.py

```diff
@@ -254,6 +280,7 @@ def create_crossref_issues_context(journal, identifiers):
     return crossref_issues
 
 
+@cache(30)
```

Comment:
On closer inspection, this cache doesn't provide much performance gain, because calls to the function are already sorted and split up by issue. In other words, the cache isn't used because the function is only ever called with unique parameters. So I will remove it.